docs/guides/migration-from-redux.md

```diff
@@ -274,6 +298,101 @@ const incrementAsync = declareAction(({ dispatch }) => {
 store.dispatch(incrementAsync())
 ```
 
+Transferring logic from sagas to effects can be very painful, but redux-saga can work with any external state management library. Just use runSaga. Redux-saga uses a [pattern](redux-saga.js.org/docs/api/#takepattern) to determine the needed action
```

Comment:
My take:

"Changing logic from sagas to effects could be painful. Redux-saga allows us to work with any external state management library through `runSaga` function. Redux-saga uses [patterns](redux-saga.js.org/docs/api/#takepattern) to determine which action is needed."

Reply:
Totally agree with you